SIRO_README.md

```diff
@@ -22,22 +22,34 @@ Project-specific README for SIRo.
 
 # Floorplanner Dataset
 
-FP is actually four distinct pieces: Floorplanner scenes, Amazon-Berkeley objects, Google Scanned objects, and Floorplanner episodes.
+FP is actually four distinct pieces: (1) Floorplanner scenes, (2) Amazon-Berkeley objects, (3) Google Scanned objects, and (4) Floorplanner episodes. (YCB objects should have been already downloaded as part of the earlier SIRo install steps)
 
-1. Download Floorplanner Scenes: [fpss_osmm.zip](https://drive.google.com/file/d/1-utUMfUbbzg_zUE5GcGNdk1UEK6lSbXe/view?usp=sharing)
+1. Download Floorplanner Scenes:
+```
+# Go to the habitat data directory
+cd habitat-lab/data
+# Make sure you have git-lfs installed
+git lfs install
+# Clone dataset, and it will take a while to finish
+git clone https://huggingface.co/datasets/fpss/fphab
+# change the name, and Floorplanner Scenes is ready to use
+mv fphab fpss
```

Comment:
How about this? I've modified instructions a bit and added sanity check to make sure complete FP scenes dataset was downloaded from the Hugging Face.
```suggestion
# 1. Go to the habitat data directory
cd habitat-lab/data
# 2. Install Git LFS from https://git-lfs.com/
# 3. Set up Git LFS for your user account
git lfs install
# 4. Clone dataset (it will take a while to finish)
git clone https://huggingface.co/datasets/fpss/fphab
# 5. Change the name, and Floorplanner Scenes is ready to use
mv fphab fpss
# 6. Sanity check (this should open FP scene in the habitat viewer)
# ./build/viewer if compiling locally
habitat-viewer --enable-physics --dataset /path/to/data/fpss/fphab/fphab.scene_dataset_config.json -- 108294897_176710602.scene_instance.json
```